bundles/org.openhab.binding.electroluxair/src/main/java/org/openhab/binding/electroluxair/internal/handler/ElectroluxAirBridgeHandler.java

```diff
@@ -147,6 +148,8 @@ private void stopAutomaticRefresh() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        return;
+        if (CHANNEL_STATUS.equals(channelUID.getId()) || command instanceof RefreshType) {
```

Comment:
Status is a read-only channel, so I assume only refresh command is intended here?
```suggestion
        if (CHANNEL_STATUS.equals(channelUID.getId()) && command instanceof RefreshType) {
```

Reply:
Correct, I'll fix!